[FIX] website: reCaptcha line alignment issues

alap-odoo opened the pull request:
Issue:
The reCAPTCHA line was taking up unnecessary space when aligned to the left.

Reason:
The line is placed inside the form block, which reserves space for a label. Since reCAPTCHA has no label, an empty space was shown.

Fix:
Hide the label by setting the visibility to none for the reCAPTCHA line.

task-4756289

robodoo commented:
[![Pull request status dashboard](https://mergebot.odoo.com/odoo/odoo/pull/225484.png)](https://mergebot.odoo.com/odoo/odoo/pull/225484)

shsa-odoo commented:
Need to change base to 19

alap-odoo commented:
> > > @alap-odoo could you rebase please
> > 
> > 
> > As per discussion [here](https://www.odoo.com/mail/message/885956951), I have already fixed it in version 17.0. However, I recently pushed the latest changes in my branch.
> 
> @alap-odoo You already fixed it in 17.0, so why is this PR necessary in 17.0? (not clear to me)

I mean to say that this PR is fixing it from 17.0. So, what's the need of rebase? Not able to get it.

bvr-odoo commented:
> > > > @alap-odoo could you rebase please
> > > 
> > > 
> > > As per discussion [here](https://www.odoo.com/mail/message/885956951), I have already fixed it in version 17.0. However, I recently pushed the latest changes in my branch.
> > 
> > 
> > @alap-odoo You already fixed it in 17.0, so why is this PR necessary in 17.0? (not clear to me)
> 
> I mean to say that this PR is fixing it from 17.0. So, what's the need of rebase? Not able to get it.

"Rebase" doesn’t mean "rebase on another branch". I just wanted you to rebase on 17.0 before merging, since it hadn’t been done for a month already.

alap-odoo commented:
> > > > > @alap-odoo could you rebase please
> > > > 
> > > > 
> > > > As per discussion [here](https://www.odoo.com/mail/message/885956951), I have already fixed it in version 17.0. However, I recently pushed the latest changes in my branch.
> > > 
> > > 
> > > @alap-odoo You already fixed it in 17.0, so why is this PR necessary in 17.0? (not clear to me)
> > 
> > 
> > I mean to say that this PR is fixing it from 17.0. So, what's the need of rebase? Not able to get it.
> 
> "Rebase" doesn’t mean "rebase on another branch". I just wanted you to rebase on 17.0 before merging, since it hadn’t been done for a month already.

Oh got it. Sorry for misunderstanding.
I have rebased it. GTG from myside.

bvr-odoo reviewed (changes requested):
The commit message needs to be updated because it still mentions your first fix.
@alap-odoo
